src/components/login/loginForm.tsx

```diff
@@ -1,73 +1,125 @@
-              <ul>
-                <li className="google">
-                  <a href="#">Google</a>
-                </li>
-                <li className="fb">
-                  <a href="#">Facebook</a>
-                </li>
-              </ul>
-            </div>
-
-            <div className="_or">or</div>
-
-            <form onSubmit={signIn} className="the-form">
-              <label htmlFor="email">Email</label>
-              <input
+      <ThemeProvider theme={defaultTheme}>
+        <Container component="main" maxWidth="xs">
+          <CssBaseline />
+          <Box 
+            sx={{
+              margintTop: 8,
+              display: 'flex',
+              flexDirection: 'column',
+              alignItems: 'center',
+            }}>
+              <Avatar sx={{ m: 1, bgcolor: 'secondary.main'}}>
+                <LockOutlinedIcon />
+              </Avatar>
+              <Typography component = "h1" variant = "h5">Sign In</Typography>
+          
+            <Box component="form" onSubmit={signIn} noValidate sx={{ mt: 1 }}>
+              <TextField 
+                margin="normal"
+                required
+                fullWidth
+                id="email"
                 type="email"
+                label="Email Address"
                 name="email"
-                id="email"
-                value={email}
-                onChange={(e) => setEmail(e.target.value)}
-                placeholder="Enter your email"
-              ></input>
-
-              <label htmlFor="password">Password</label>
-              <input
-                type="password"
+                autoComplete="email"
+                autoFocus 
+              />
+              <TextField 
+                margin="normal"
+                required
+                fullWidth
                 name="password"
+                label="Password"
+                type="password"
                 id="password"
-                value={password}
-                onChange={(e) => setPassword(e.target.value)}
-                placeholder="Enter your password"
-              ></input>
-
-              <input type="submit" value="Log In"></input>
-            </form>
-          </div>
-
-          <div className="form-footer">
-            <div>
-              <span>Don't have an account?</span> <a href="">Sign Up</a>
-            </div>
-          </div>
-        </div>
-      </div>
+                autoComplete="current-password"
+              />
+              <FormControlLabel
+                control={<Checkbox value="remember" color="primary" />}
+                label="Remember me" 
+              />
+              <Button
+                type="submit"
+                fullWidth
+                variant="contained"
+                sx={{ mt: 3, mb: 2 }}
+              >
+                Sign In
+              </Button>
+              <Grid container>
+              <Grid item xs>
+                <Link href="#" variant="body2">
```

Comment:
Can you make sure this 'forgot password' link works? I'm sure we can use Firebase Authentication for this. However, you'll need to make a function that the links onClick can use.

Reply:
forgot password is now complete. The user receives and email allowing them to change their password